main.cpp

```diff
@@ -23,7 +23,7 @@ int main(void)
 {
 #if MBED_CONF_APP_WAIT_FOR_SYNC
     tfm_log_printf("Waiting for Greentea host\n");
-    GREENTEA_SETUP(60, "default_auto");
```

Comment:
Commit message - `Increase timeout to 600 seconds`

Reply:
Done